quality_of_life_changer.client/src/redux-modules/user-profile/constants.js

```diff
@@ -1,4 +1,8 @@
 export const userProfileConstants = {
   GET_USER_CALENDARS: "GET_USER_CALENDARS",
   SET_USER_CALENDARS: "SET_USER_CALENDARS",
+
+  DELETE_CALENDAR: "DELETE_CALENDAR",
+
```

Comment:
why do you need this gap?
better divide actions here to 'actions for reducers' and 'actions for sagas'. you can even add corresponding comments in code.